src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -99,6 +101,8 @@ public void onChatMessage(ChatMessage chatMessage) {
       showNotificationIfNecessary(chatMessage);
       setUnread(true);
       incrementUnreadMessagesCount(1);
+      boolean chatFocused = preferencesService.getPreferences().getMainWindow().getLastView().equals(NavigationItem.CHAT.name());
+      eventBus.post(new UnreadPrivateMessageEvent(chatMessage, chatFocused));
```

Comment:
The event should only say "hey, there's a new private message", whether or not chat is focused should not be part of the event, but rather checked by the receiver (if of interest). Also, while checking the value in the preferences might work, it's semantically incorrect because we are not interested in the preferences value but in the current application state. It's to be considered a coincidence that the preferences reflect the current application state.
So the receiver of the event should have some different mechanism to figure out whether chat is active or not.